src/components/MyNdla/AddResourceToFolder.tsx

```diff
@@ -285,11 +284,12 @@ const AddResourceToFolder = ({ onClose, resource, defaultOpenFolder }: Props) =>
         >
           {t("cancel")}
         </Button>
-        <LoadingButton
+        <Button
```

Comment:
I tilfeller som dette stemmer jeg for at vi bare fortsetter å bruke disabled som vi gjorde før, bare at vi dropper `addResourceLoading`-delen

Reply:
Så det skal ikke være loading button?

Reply:
Dårlig ordlagt av meg! Joda, det skal være loading. Bare bruk vanlig disabled-prop, men ikke inkluder `addResourceLoading` i `disabled`-sjekken

Reply:
jupp det er allerede fjernet fra disabled-sjekk